cfgov/jobmanager/signals.py

```diff
@@ -0,0 +1,31 @@
+import logging
+import requests
+
+from flags.state import flag_enabled
+
+from wagtail.wagtailcore.signals import page_published
+
+from jobmanager.models.pages import JobListingPage
+
+logger = logging.getLogger(__name__)
+
+
+SITEMAP_URL = 'https://www.consumerfinance.gov/sitemap.xml'
+GOOGLE_URL = 'http://www.google.com/ping'
+
+
+def request_site_recrawl(sender, **kwargs):
+    try:
+        if flag_enabled('PING_GOOGLE_ON_PUBLISH'):
+            response = requests.get(GOOGLE_URL, {'sitemap': SITEMAP_URL})
+            response.raise_for_status()
+            logger.info(
+                'Pinged Google after job page publication.'
+            )
```

Comment:
@chosak Your flag/exception suggestions should be implemented in https://github.com/cfpb/cfgov-refresh/pull/3784/commits/b887abea003ceb013f54530e17244afd33f3b1d9. Thank you!